### PR TITLE
Feature/fetch users data

### LINKED ISSUE #5: Fetch All User Data

### LINKED ISSUE #8: Filter Trips By userID

### PR DESCRIPTION
**Pull request by holladayian**
## Description
This PR adds two fetch methods to fetch the userData and their respective userTripData

## Context
This PR is smaller than I usually do, but upon eval of last project, we were told to add a lot more PRs, hence the shortness.
The context of the actual code is fetching user data.

## Where To Start
I have a eventlistener in `index.js` that fires `fetchUser` in `fetch.js`, which in turn calls `fetchTripsForAUser`.
For now this will console.log a userTripData.
This isn't fully fleshed out, but as I said before, I am trying to make smaller PRs, and thus, getting these two fetch methods seemed concise enough to pull. 

## Affected areas of application
So far all this does is console.log data. Next step will be passing that data into functions that can filter by needed "filters".

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] Test A: console checkouts

## Relevant Tickets
#6 
Fixes #8 
Fixes #5 

## Tags
@BrigetteDoelp 

## Questions or Notes (if applicable):
I think the order of opps goes, user-interaction -> index.js(call function on fetch.js) -> fetch.js(call function in DOM-updates) -> DOM-updates.js (update the dom)

### CODE REVIEW

File: src/fetch.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+let fetcher = {
```
Comment (BrigetteDoelp) [inline src/fetch.js:1]: This looks good! Really convenient setup having the `fetch` stuff wrapped in an object.

**Comment by BrigetteDoelp** [comment]
Also, this *is* a very small PR. When you got the feedback from your last project, how large were the PRs? 

This is just my opinion, that of a student, but it would make more sense to me to have one PR for getting ALL the fetch stuff running? Then again, maybe that would be too large of a PR based on your previous feedback.

**Comment by holladayian** [comment]
We ended up having 11 PRs, and were told we should have closer to 50. Bob said our PRs should really only close 2-3 tickets (depending)

**Comment by BrigetteDoelp** [comment]
Well alright then! I'd say this looks great, and I will now reevaluate the size of my PRs. I'll go ahead and merge!

**Pull request merged**